index.html

```diff
@@ -0,0 +1,1011 @@
+                                <span>on VentureBeat.</span></a>
+                        </div>
+                    </div>
+                </div>
+                <div class="lf-content lf-4">
+                    <div class="lf-titles">
+                        <a href="#">
+                            <img src="./img/download.png" alt="" width="60" height="60" class="small-img">
+                        </a>
+                        <h3>Ringover</h3>
+                    </div>
+                    <div class="lf-subtitle">
+                        <p class="lf-subtitle-price">
+                            $12M
+                        </p>
+                        <p class="lf-subtitle-serie">
+                            SERIES A
+                        </p>
+                    </div>
+                    <div class="lf-color">
+                        <div class="lf-body">
+                            <a href="#" class="lf-body-links">
+                                <i class="fas fa-tag"></i>
+                                Telecommunications
+                            </a>
+                            <a href="#" class="lf-body-links">
+                                <i class="fas fa-briefcase"></i>
+                                Expedition Growth Capital
+                            </a>
+                        </div>
+                        <div class="lf-footer">
+                            <a href="#">Ringover raises $12 million to bring enterprise communications to the cloud
+                                <span>on VentureBeat.</span></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Categories -->
+        <div class="categories-container">
+            <div class="categories-content">
+                <div class="categories-title">
+                    <h3>Hard Fork <i class="fas fa-chevron-right"></i></h3>
+                </div>
+                <div class="categories-img ct-img-1">
+                    <div class="categories-img-txt">
+                        <h4>MOST POPULAR</h4>
+                        <a href="#" class="ct-img-txt-main">
+                            GameStop stocks hit record high — thanks to Redditors
+                        </a>
+                        <p class="ct-img-txt-sub">
+                            <a href="#">Ivan Mehta</a> - 1 day ago
+                        </p>
+                    </div>
+                </div>
+                <div class="categories-subtitle-img">
+                    <div class="subtitle-left">
+                        <p class="subtitle-tile">LATEST</p>
+                        <p class="subtitle-txt">
+                            Alibaba shares jump 8% as Jack Ma surfaces after 3 months
+                        </p>
+                    </div>
+                    <div class="subtitle-right ct-sub1">
+                    </div>
+                </div>
```

Comment:
For each category in the `Categories` section, the second article should have a clickable (`a` tag) image on the right, and a title on the left that uses a `hx` tag and a link (`a` tag)